src/plugins/sentryPlugin.ts

```diff
@@ -18,6 +17,16 @@ export const sentryPlugin: ApolloServerPlugin = {
                  to request-specific lifecycle events. */
     return {
       async didEncounterErrors(ctx) {
+        //for error codes:
+        // https://www.apollographql.com/docs/apollo-server/data/errors/#bad_user_input
+        const errorCodes = [
```

Comment:
didn't include persisted_query errors as the responds are sent before plugin code is processed